### ISSUE TITLE
Feature Request: Android, Generate Round Icons

### ISSUE THREAD

**Issue by puremana** [bamlab/react-native-make#26]
At the moment using the `set-icon` command, you are unable to generate a different, rounded version of the icon, such as creating another icon file with the name 'ic_launcher_round' in all minmap folders.

Using this package, since no rounded image is set in the manifest, new android versions (android version 10 for instance) will crop the default icon into a circle, often not looking very professional.

One temporary solution until the ability to generate a rounded icon is introduced, is to alter the AndroidManifest and add `android:roundIcon="@mipmap/ic_launcher"` which means android will no longer crop the icon and instead just display your default android icon. Since your default icon can already be slightly rounded and be transparent, this looks fine.

**Comment by yleflour** [comment]
In the current situation we expected the input file to be used for any kind of icon. 

Would be willing to look into differentiating square / rounded if you have a proposal. What would you expect in terms of CLI usage without breaking existing commands and params?

**Comment by puremana** [comment]
> Would be willing to look into differentiating square / rounded if you have a proposal. What would you expect in terms of CLI usage without breaking existing commands and params?

Having something like `--round` as an optional parameter on `set-icon` which if true, generates the `ic_launcher_round` files would be fine. Or, you could have an optional `--iconname` parameter which defaults to `ic_launcher` but would allow someone to enter in `ic_launcher_round` if they wanted to and then just do the manifest changes themselves. Either would have worked in my case. 

Thanks for replying and looking into this so quickly by the way!

**Comment by adibas03** [comment]
@puremana do you think thi sis related to #14

**Comment by puremana** [comment]
> @puremana do you think this is related to #14

Yes. Sorry I didn't see that issue before

**Comment by blackinitial** [comment]
yeah we need generate round icons, my icon app showing default icon on launcher
hmm..

**Comment by muhsin-k** [comment]
Any update on this?

**Comment by rahamin1** [comment]
How about creating the icons for android using https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html ?

**Comment by yleflour** [comment]
Hey, sorry for the long silence. I currently don't have time to manage this library, the situation might evolve in the upcoming months but nothing I can commit myself to.

Regarding icons generation, I think the best way to do it is simply to open the android folder with Android Studio and use the embedded icon tool. I haven't been able to create a tool myself that matches its features and it's easy enough that I believe the Android Icon feature should be deprecated.

What do you think about this ?
